tests/sccache_cargo.rs

```diff
@@ -91,8 +87,11 @@ fn test_rust_cargo_cmd(cmd: &str) {
         .success();
     // `cargo clean` first, just to be sure there's no leftover build objects.
     let envs = vec![
-        ("RUSTC_WRAPPER", &sccache),
-        ("CARGO_TARGET_DIR", &cargo_dir),
+        ("RUSTC_WRAPPER", sccache.as_ref()),
+        ("CARGO_TARGET_DIR", cargo_dir.as_ref()),
+        // Explicitly disable incremental compilation because sccache is unable
+        // to cache it at the time of writing.
+        ("CARGO_INCREMENTAL", OsStr::new("0")),
```

Comment:
now I'm lost

Reply:
The correct expected cache hits is 2 with disabled CARGO_INCREMENTAL. This is what we had in our CI setup which is why the results did differ between local runs (where incremental is enabled by default) and the CI. This unconditionally disables incremental in the test run so that it's independent from the environment it's run in - be it CI or locally.

Reply:
gotcha, thanks for the explanation!